WebApp/autoreduce_webapp/test/oldtest_reduction_variables.py

```diff
@@ -1,4 +1,4 @@
-import logging, os, sys, shutil, imp, json
+import logging, os, sys, json
```

Comment:
Might be worth commenting at the top of the old test classes to say that they need to be assessed for validity/usefulness - just for clarity.